rpmfile/__init__.py

```diff
@@ -108,13 +152,31 @@ def __exit__(self, *excinfo):
 
     _members = None
 
+    @staticmethod
+    def _resolve_links(ino_map):
+        for _, _ino_members in ino_map.items():
+            _links_or_dirs = []
+            target_member = None
+            for member in _ino_members:
+                if member.size > 0 and not member.isdir:
```

Comment:
Possible bug if this branch gets True more than once. if not possible, add assertion